web/admin/src/Admin.js

```diff
@@ -201,6 +221,36 @@ export default class Admin extends PureComponent {
     )
   }
 
+  getAdmins = keys => {
+    const attendeeClickPromises = keys.map(result =>
+      client
+        .getAttendee(result)
+        .then(attendee => ({ ...attendee }))
+        .catch(err => result),
```

Comment:
This means that if an attendee is not found, instead of resolving to that attendee object, it will resolve to a string (the attendee's ID).  Are you sure this is desired?